packages/nc-gui-v2/middleware/auth.global.ts

```diff
@@ -33,6 +33,9 @@ export default defineNuxtRouteMiddleware((to, from) => {
 
   /** if auth is required or unspecified (same as required) and user is not signed in, redirect to signin page */
   if ((to.meta.requiresAuth || typeof to.meta.requiresAuth === 'undefined') && !state.signedIn.value) {
+    if (state.appInfo?.value?.firstUser) {
```

Comment:
change to just `if (state.appInfo.value.firstUser)`

`?` still suggests that this might be undefined - it has to be defined at this stage or our app is working incorrectly.
`appInfo` is fetched inside a plugin, which has to run before the middleware.